dv/uvm/core_ibex/scripts/compile_tb.py

```diff
@@ -19,23 +21,51 @@
 logger = logging.getLogger(__name__)
 
 
-def _get_iss_pkgconfig_flags(specifiers, iss_pc, simulator):
+def _get_iss_pkgconfig_flags(specifiers: List[str], iss_pc: List[str], simulator: str) -> str:
+    all_tokens = []
+
     # Seperate pkg-config calls for each specifier as combining them has been
     # observed misbehaving on CentOS 7
-    _flags = ' '.join([subprocess.check_output(
-                        args=(['pkg-config', s] + iss_pc),
-                        universal_newlines=True,).strip()
-                       for s in specifiers])
-
-    if simulator == 'xlm':
-        # See xcelium documentation for the -Wld syntax for passing
-        # flags to the linker. Passing -rpath,<path> options is tricky
-        # because commas are parsed strangely between xrun and the xmsc
-        # tool, and its easy for the options to arrive malformed. Use
-        # the following hack to get it through.
-        if '-Wl' in _flags:  # This should be in LDFLAGS only
-            _flags = "'-Xlinker {}'".format(_flags.replace('-Wl,', ''))
-    return _flags
+    # Generate a list of tokens for each call, and append it to the all_tokens variable
+    for s in specifiers:
+        cmd = ['pkg-config', s] + iss_pc
+        stdout = subprocess.check_output(cmd, universal_newlines=True).strip()
+        tokens = shlex.split(stdout)
+        logger.debug(f"pkgconfig_tokens = {tokens}")
```

Comment:
I'm not sure I know enough about Python logging to have an opinion about this: do you want to keep these debug prints?